### PR TITLE
Make badge numbers configurable for round-hp theme

### PR DESCRIPTION
**Pull request by mmellado**

### CODE REVIEW

File: themes/round-hp/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ - **HP** is shown around the Pokémon and is updated dynamically
+ - **HP Color** HP color changes at 50% and 20%
+ - **Nickname** is shown in the box below each
+ - **Level** Level is also shown below the Pokémon nickname
+ - **Background** Background color is displayed using the Pokémon's type
+
+## Settings
+
+### Pokemon Gen 7 Art
+This theme relies on Gen 7 Art provided by PokeLink. However the sprites can be updated to other PokeLink supported sprites by updating the `imgPahts` in `./assets/js/themeSettings.js`
```
Comment (NoelDavies) [inline themes/round-hp/README.md]: "paths" typo

**Review by NoelDavies — APPROVED** [review]

**Comment by xLink** [comment]
I know its not a huge thing, but a screenie for what a party looks like with this theme would be advantagious,

**Comment by mmellado** [comment]
I thought that was already there in the preview?

**Comment by xLink** [comment]
i mean in here, we can pull it and preview ourselves, it just saves the extra step, and also gives us a preview before it gets merged etc

**Comment by mmellado** [comment]
ahh you mean in the PR itself?

**Comment by mmellado** [comment]
Preview of theme

preview

**Comment by mmellado** [comment]
The image with this preview is also available in the theme's `assets/img/preview.png`

**Pull request merged**